cost-analyzer/values.yaml

```diff
@@ -671,3 +671,4 @@ serviceAccount:
 #    key: ""
 #    enabled: false
 #    secretname: productkeysecret # create a secret out of a file named productkey.json of format { "key": "kc-b1325234" }
+#  multiCloudSecret: "multi-cloud-config"
```

Comment:
Nice! 

Should this this be more generic, i.e. `cloudIntegrationSecret`, and then make this the recommended path going forward for all integrations? 

Have docs yet on how this can/should be used?

Reply:
Currently the method of using this secret is separated in such a way that it would be easy to feature gate it. Some things could be moved around to make that more possible, I could definitely change up the name here though. 

I have a start on the docs, but they are not ready yet.